skimage/transform/_geometric.py

```diff
@@ -910,9 +910,9 @@ def __init__(self, matrix=None, scale=None, rotation=None, shear=None,
                 sx, sy = scale
 
             self.params = np.array([
-                [sx * math.cos(rotation), -sy * math.sin(rotation + shear), 0],
-                [sx * math.sin(rotation),  sy * math.cos(rotation + shear), 0],
-                [                      0,                                0, 1]
+                [sx * math.cos(rotation), -sy * (math.tan(shear) * math.cos(rotation) + math.sin(rotation)), 0],
+                [sx * math.sin(rotation), -sy * (math.tan(shear) * math.sin(rotation) - math.cos(rotation)), 0],
+                [                      0,                                                                 0, 1]
             ])
             self.params[0:2, 2] = translation
```

Comment:
How about reformatting this a little. I think 
```python
            a0 = sx * math.cos(rotation)
            a1 = -sy * (
                math.tan(shear) * math.sin(rotation) + math.sin(rotation)
            )
            a2 = translation[0]
            b0 = sx * math.sin(rotation)
            b1 = -sy * (
                math.tan(shear) * math.sin(rotation) - math.cos(rotation)
            )
            b2 = translation[1]
            self.params = np.array([[a0, a1, a2], [b0, b1, b2], [0, 0, 1]])
```
may be  a bit easier to follow. (GitHub won't let me use the suggest feature here)

Reply:
Thanks, I fixed this.